src/Compute/Compute/Extension/AzureDiskEncryption/SetAzureDiskEncryptionExtension.cs

```diff
@@ -571,11 +576,24 @@ public override void ExecuteCmdlet()
                     //          updates VM 
 
                     // First Pass
-                    AzureOperationResponse<VirtualMachineExtension> firstPass = this.VirtualMachineExtensionClient.CreateOrUpdateWithHttpMessagesAsync(
-                        this.ResourceGroupName,
-                        this.VMName,
-                        this.Name,
+
+                    AzureOperationResponse<VirtualMachineExtension> firstPass;
```

Comment:
This is another exampel that liekly will not work.  The commands that occur later in execution depend on the first command having successfully completed

Reply:
Agreed. Done.